src/components/profile/ViewOrUploadAvatar/index.js

```diff
@@ -120,25 +125,61 @@ ViewOrUploadAvatar.navigationOptions = {
   title: TITLE,
 }
 
-const getStylesFromProps = ({ theme }) => ({
-  section: {
-    flex: 1,
-    position: 'relative',
-  },
-  cameraButton: {
-    left: 'auto',
-    position: 'absolute',
-    right: 12,
-    top: theme.sizes.defaultDouble,
-  },
-  closeButton: {
-    left: 12,
-    position: 'absolute',
-    top: theme.sizes.defaultDouble,
-  },
-  doneButton: {
-    marginTop: 'auto',
-  },
-})
+const getStylesFromProps = ({ theme }) => {
+  const { defaultDouble, defaultQuadruple } = theme.sizes
+  const buttonGap = getDesignRelativeWidth(-30) / 2
+
+  return {
+    section: {
+      flex: 1,
+      position: 'relative',
+      alignItems: 'center',
+      justifyContent: 'center',
+    },
+    cameraButtonContainer: {
+      zIndex: 1,
+    },
+    cameraButton: {
```

Comment:
@mauriguz layout changes that are not Platform.select needs to be verified are still looking good on web

Reply:
All these changes had been tested before pushing, most of these styles were also taken from the same component in master